### DISCUSSION TITLE
Waybar Customizations?

### DISCUSSION THREAD

**Discussion by tacosrdelicioso** [prasanthrangan/hyprdots#322] [Q&A]
His waybar config seems really complicated and everytime I've tried to touch it to add stuff it breaks and explodes. I'm also trying to get rid of the blur across the entire bar...I  prefer the clean look of just the modules on the bar

**Comment by T-Crypt** [comment]
You need to remove this from the hyprland theme configurations.

Directory: `.config/hypr/themes/`

for each theme in themes 

remove line 
`blurls = waybar`

SEARCH Results: https://github.com/search?q=repo%3Aprasanthrangan%2Fhyprdots%20blurls%20%3D%20waybar&type=code

**Reply by tacosrdelicioso** [reply]
Thanks boss. I was trying to do it from waybar config because there it looked like he had a waybar bg setup for the blur. This config is 10x more complicated than my previous hyprland setup so...I had one hyprland.conf for everything and waybar etc weren't all affected by a global theme so..bit of a culture shock

**Reply by tacosrdelicioso** [reply]
> You need to remove this from the hyprland theme configurations.
> 
> 
> 
> Directory: `.config/hypr/themes/`
> 
> 
> 
> for each theme in themes 
> 
> 
> 
> remove line 
> 
> `blurls = waybar`
> 
> 
> 
> SEARCH Results: https://github.com/search?q=repo%3Aprasanthrangan%2Fhyprdots%20blurls%20%3D%20waybar&type=code

So for some of them blurls is not what's doing the blur and it is using a bg of some sort. Eg decay-green
